cabal.project

```diff
@@ -1,2 +1,7 @@
-packages:
-    ./
+packages: .
+
+source-repository-package
+  type: git
+  location: https://github.com/prolic/secp256k1-schnorr.git
+  tag: 166b0d27d79e732731a54c7ec8e27e8f939cb6f5
+  --sha256: sha256-5EBiOcLOujfwyTYp0oLb26RDqC89ucjIDYx+c5GZmrk
```

Comment:
This is needed so that haskell.nix knows the hash to expect. If you bump the tag, change one of the characters in here to an `A` or something so that rebuilding fails; that will tell you the actual sha256 to use.